pagerduty/event_orchestration_path_test.go

```diff
@@ -172,34 +172,37 @@ func TestEventOrchestrationPathRouterPathUpdate(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	want := &EventOrchestrationPath{
-		Type: "router",
-		Parent: &EventOrchestrationPathReference{
-			ID:   "E-ORC-1",
-			Self: "https://api.pagerduty.com/event_orchestrations/E-ORC-1",
-			Type: "event_orchestration_reference",
-		},
-		Sets: []*EventOrchestrationPathSet{
-			{
-				ID: "start",
-				Rules: []*EventOrchestrationPathRule{
-					{
-						Actions: &EventOrchestrationPathRuleActions{
-							RouteTo: "P3ZQXDF",
-						},
-						Conditions: []*EventOrchestrationPathRuleCondition{
-							{
-								Expression: "event.summary matches part 'orca'",
+	want := &EventOrchestrationPathPayload{
```

Comment:
Here and below modified expected data to be  of the `EventOrchestrationPathPayload` type instead of `EventOrchestrationPath` to reflect changes to the Update endpoint